From 86f8e96c3d0ed6d4380576719be11e2a69937711 Mon Sep 17 00:00:00 2001
From: t00thpick1 <t00thpick1dirko@gmail.com>
Date: Mon, 14 Apr 2014 21:35:01 -0400
Subject: [PATCH] Re-add missed diff in 1.7.5 update. Fixes BUKKIT-5536

In the 1.7.5 update the diff that called book edit events when editing books
was accidentally dropped because of nms changes within the file.  This commit
re-adds the craftbukkit call to restore event behavior.
---
 src/main/java/net/minecraft/server/PlayerConnection.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/net/minecraft/server/PlayerConnection.java b/src/main/java/net/minecraft/server/PlayerConnection.java
index 5c1d17a..27036b2 100644
--- a/src/main/java/net/minecraft/server/PlayerConnection.java
+++ b/src/main/java/net/minecraft/server/PlayerConnection.java
@@ -1710,7 +1710,7 @@ public class PlayerConnection implements PacketPlayInListener {
                 itemstack1 = this.player.inventory.getItemInHand();
                 if (itemstack1 != null) {
                     if (itemstack.getItem() == Items.BOOK_AND_QUILL && itemstack.getItem() == itemstack1.getItem()) {
-                        itemstack1.a("pages", (NBTBase) itemstack.getTag().getList("pages", 8));
+                        CraftEventFactory.handleEditBookEvent(player, itemstack); // CraftBukkit
                     }
 
                     return;
-- 
1.8.3.2

